src/pages/getting-started/index.mdx

```diff
@@ -2,33 +2,57 @@
+
+Your customers can start multiple conversations with agents, as well as continue the existing ones - it's up to them. Keeping track of chat history is easy, as agents can scroll up to get a preview of all the exchanged messages in a conversation.
+
+All of these functionalities are guaranteed _out of the box_, but this is just a tiny bit of what LiveChat offers. If you're interested in other default features, see the <a href="https://www.livechat.com/features/" target="_blank">full list of LiveChat features</a>.
+
+## What LiveChat can be
+  
+We're constantly trying to come forward with the new ways customers could integrate with LiveChat. Certain business needs call for specific customization, whether it's creating an app or just tweaking the look & feel.
+
+### Extending LiveChat
+
+LiveChat can extend its interface by building apps to add new panels, widgets, and UI elements. When integrating with the Agent App, developers can choose from various options. All options were designed to extend different app locations, making it very flexible. Our Platform offers, for example, apps in the Details section (widgets), fullscreen apps displayed in the main menu, or apps in the Customer Details section (Customer Detail widgets). <a href="https://developers.livechat.com/docs/extending-agent-app" target="_blank">Learn more...</a> The <a href="https://developers.livechat.com/docs/extending-agent-app/products-sdk/agent-app-sdk" target="_blank">Agent App SDK</a> is currently the primary way to develop your own widgets.
+
+Just as there are ways to build upon the App, there are also some to extend the [Chat Widget](#chat-widget) using <a href="https://developers.livechat.com/docs/extending-chat-widget/javascript-api/" target="_blank">Chat Widget JavaScript API</a> or <a href="https://developers.livechat.com/docs/extending-chat-widget/customer-sdk/" target="_blank">Chat Widget Customer SDK</a>.
 
-# What HelpDesk is
+Besides extending LiveChat UI, you can also develop LiveChat using our [Messaging](#messaging) and [Management](#management) APIs or by [interacting with data](#interacting-with-data).
+
+### Monetizing your LiveChat app
+
+After building your LiveChat app, personalizing it, and fitting it to your customers' needs, you can [monetize](#monetization) it and place it on our <Placeholder id="LIVECHAT_MARKETPLACE_URL"/> to reach many potential customers and gain some profit.
+
+💡 If you want to read about building LiveChat apps, go to our <a href="https://developers.livechat.com/docs/getting-started/helpdesk-apps" target="_blank">tutorial</a>.
+
+# Meet HelpDesk
 
 <a href="https://www.helpdesk.com/" target="_blank">HelpDesk</a> is a ticketing system that helps you to manage both your asynchronous and synchronous communication with customers. You can imagine it as a mailbox with many features that make communication much more manageable:
```

Comment:
I'm not sure about how HD helps with synchronous communication. Via the HD+LC integration?

Reply:
Ajajajaj, I had to mix something, I was taking it from their website, and when I look at it now, you're right it's asynchronous communication. My mistake